appcheck/app/src/main/java/com/google/firebase/example/appcheck/kotlin/ApiWithAppCheckExample.kt

```diff
@@ -20,14 +23,19 @@ class ApiWithAppCheckExample {
         .build()
         .create(YourExampleBackendService::class.java)
 
-    fun callApiExample() {
-        FirebaseAppCheck.getInstance()
-            .getAppCheckToken(false)
-            .addOnSuccessListener { tokenResponse ->
-                val appCheckToken = tokenResponse.token
-                val apiCall = yourExampleBackendService.exampleData(appCheckToken)
-                // ...
-            }
+    suspend fun callApiExample() {
+        val tokenResponse = Firebase.appCheck.getToken(false).await()
+        val appCheckToken = tokenResponse.token
+        val apiCall = yourExampleBackendService.exampleData(appCheckToken)
+        // ...
     }
 }
 // [END appcheck_custom_backend]
+
+class Misc {
+    private suspend fun getLimitedUseToken() {
+        // [START appcheck_get_limited_use_token]
+        val tokenResponse = Firebase.appCheck.limitedUseToken.await()
```

Comment:
I think none of the Kotlin snippets we currently have on devsite are using `await()` yet. :/ Can we use the listener here (and in `callApiExample`)?

I'd like to keep it consistent for and we can update them all at once.